src/control/server/ctl_system.go

```diff
@@ -200,20 +200,71 @@ func (svc *ControlService) SystemStop(ctx context.Context, req *ctlpb.SystemStop
 	return resp, nil
 }
 
+// restartHarness restarts ranks managed by harness running on remote host.
+// TODO: specify specific rank(s) to restart in request
+func (svc *ControlService) restartHarness(ctx context.Context, leader *IOServerInstance, addr string) error {
+	_, err := leader.msClient.Restart(ctx, addr, &mgmtpb.RestartRanksReq{})
+
+	return err
+}
```

Comment:
Might be more accurate to name this `startRemoteHarness()`. Also you could add ranks as a variadic param. If none are supplied, that means start all ranks.